src/types.ts

```diff
@@ -55,6 +55,36 @@ export interface Response {
   data: any
 }
 
+export interface ScalarDefaults {
+  [type: string]: string
+}
+
+export interface FilterMappings {
+  [type: string]: {
+    [operator: string]: any
+  }
+}
+
+// Operator Specific Filters
+//
+// See https://github.com/graphile-contrib/postgraphile-plugin-connection-filter#operators
+
+export interface Contains {
+  contains: any
```

Comment:
I wanted to make the values have a more narrow type, but the `type` is passed in at runtime to `createFilter`. The `type` determines how the operator is supposed to be formatted.

We can look at creating operators per type. For example:

```ts
interface ArrayContains<T> {
   contains: T[]
}

interface InternetAddressContains {
   contains: NetAddr
}
```

If we decide to go that route, then we need to figure out what to do if the `type` parameter in `createFilter` does not match the type defined in the operator.